docker/docker-test.yml

```
@@ -0,0 +1,42 @@
+      retries: 5
+      start_period: 30s
+    networks:
+      - test-network
+
+volumes:
+  mongodb_test_data:
+    driver: local
+
+networks:
+  test-network:
+    driver: bridge 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)